android/build.gradle

```diff
@@ -109,6 +109,13 @@ static def getClientId(rootProject) {
     return rootProject.clientId
 }
 
+static def getAssetVersion(rootProject) {
+    if(!rootProject.hasProperty('hyperAssetVersion')) {
+        return ""
```

Comment:
This won't work. If you send "" build will fail.

Reply:
What should be the default value then?